src/renderers/webgl/WebGLRenderLists.js

```diff
@@ -159,6 +159,7 @@ function WebGLRenderList() {
 	}
 
 	return {
+		renderItems: renderItems,
```

Comment:
I'm not sure we had this before (meaning making something public so it is accessible for unit tests).

@mrdoob Do we need to update the TS declaration file in this case?

Reply:
Hmm... I feel like it's worth it.

Reply:
I thought about this tonight and have a concern. 

Unit tests should focus on testing the public interface of a component. It's not ideal if internal have to be exposed in order to write "better" unit tests. In general, unit tests should make no assumption about the internals of a component. Otherwise you have to refactor unit tests when internals change, even if the public interface stays the same.

Hence, I think it's better to not expose `renderItems` and `WebGLRenderList`.

Reply:
> Unit tests should focus on testing the public interface of a component. It's not ideal if internals have to be exposed in order to write "better" unit tests.

Generally I agree but in this case `finish` can't really be tested otherwise because it has important behavior that otherwise isn't testable from the outside the class -- it's intended to remove references to objects so they can be garbage collected and I think it's valuable to make sure that's working right. If you have other suggestions on how to check that otherwise I'm happy to try another approach!

> > @mrdoob Do we need to update the TS declaration file in this case?
>
> I don't know. @gkjohnson what do you think?

I think the typescript definitions files are only really useful for classes and members that we expect people expect people to use publicly (it's really just useful for autocomplete and typescript error reporting) so I don't think we need to include it here.

Reply:
How about we make these methods public temporarily?
And we add comments in the code to communicate that.

Reply:
> How about we make these methods public temporarily?

Can you explain what you mean by that? "Temporarily" as in "we'll make them public and see how it goes" and maybe remove them later? Either way I'll add `WebGLRenderList` and `renderItems` to the typescript definitions and add a comment saying they were made public for the sake of testing.

Reply:
We're making them public so you can get `Group.renderOrder` fixed. Once it's fixed we can make them private again.

Reply:
So you want to remove the tests for the `finish` function later, then? To me it makes sense to keep those tests to make sure that `finish` function doesn't silently break in the future but I'm supportive either way. I've gone ahead and added the comment added `renderItems` to the typescript definitions.